src/renderer/entities/proxy/api/pureProxiesService.ts

```diff
@@ -0,0 +1,14 @@
+import { GraphQLClient } from 'graphql-request';
+
+import { CHECK_PURE_PROXIES } from './graphql/queries/pureProxies';
+import type { AccountId } from '@shared/core';
+
+export const checkPureProxies = async (client: GraphQLClient, accountIds: AccountId[]): Promise<AccountId[]> => {
+  const data = await client.request(CHECK_PURE_PROXIES, {
```

Comment:
No need for multiline 
```suggestion
  const data = await client.request(CHECK_PURE_PROXIES, { accountIds });
```